spec/emoji_char_spec.rb

```diff
@@ -30,6 +30,7 @@
       @usflag    = EmojiChar.new({'unified' => '1F1FA-1F1F8'})
       @hourglass = EmojiChar.new({'unified' => '231B', 'variations' => ['231B-FE0F']})
       @cloud     = EmojiChar.new({'unified' => '2601', 'variations' => ['2601-FE0F']})
+      @ear       = EmojiChar.new({'unified' => '1F442', 'skin_variations' => { '1F442-1F3FF' => { 'unified' => '1F442-1F3FF' }}})
```

Comment:
For these, I like to mock what we would actually get when parsing emojidata library, so although its going to be verbose we should do this with the full identical list of variations we get for an ear when importing the vendored data.